django/contrib/admin/options.py

```diff
@@ -109,6 +109,7 @@ class BaseModelAdmin(six.with_metaclass(forms.MediaDefiningClass)):
     formfield_overrides = {}
     readonly_fields = ()
     ordering = None
+    orderable_by = ()
```

Comment:
I wonder if allowing users to specify `orderable = ()` would be something worth supporting. If the answer is yes then I suppose we should be initializing this to None so we can distinguish that use case.